spec/operators/multicast-spec.ts

```diff
@@ -51,7 +51,9 @@ describe('Observable.prototype.multicast', () => {
 
   it('should accept selectors to factory functions', () => {
     const source =      hot('-1-2-3----4-|');
-    const sourceSubs =     ['^           !'];
+    const sourceSubs =     ['^           !',
+                            '    ^       !',
+                            '        ^   !'];
     const multicasted = source.multicast(() => new Subject(),
       x => x.zip(x, (a, b) => (parseInt(a) + parseInt(b)).toString()));

```

Comment:
In here, can we assert that `x` is something expected? Also, can we return something more clear? Maybe a simple `map` like: `x.map(v => v + '!')`. Then just use the values argument to `expectObservable` to assert.